taichi/transforms/lower_ast.cpp

```diff
@@ -82,6 +82,21 @@ class LowerAST : public IRVisitor {
     }
   }
 
+  void visit(FrontendFuncCallStmt *stmt) override {
+    auto block = stmt->parent;
```

Comment:
nit: Use the types here. `auto` doesn't deduce references and if the underlying type is a simple pointer it's easier to read & less likely to mess up